github/event_types.go

```diff
@@ -486,7 +486,7 @@ type PullRequestEvent struct {
 	// the pull request was closed with unmerged commits. If the action is "closed"
 	// and the merged key is true, the pull request was merged.
 	Action      *string      `json:"action,omitempty"`
-	Number      *int64       `json:"number,omitempty"`
+	Number      *int         `json:"number,omitempty"`
```

Comment:
@kshitij10496 You'll need to re-run `go generate` after this change to make CI pass.

Reply:
Thanks for the heads up.